homeassistant/components/zwave_js/services.py

```diff
@@ -441,7 +443,8 @@ async def async_multicast_set_value(self, service: ServiceCall) -> None:
             client,
```

Comment:
I'd use keywords for all parameters to make the call more clear.

Reply:
How would I do that? Would I have to add an intermediary service handler that breaks out the parameters into the function call?

Reply:
No, just use keywords here in the call. Python will handle that.
```suggestion
            client=client,
```
Etc.